test/mock-objects/mock-organisation.js

```diff
@@ -0,0 +1,16 @@
+const rawOrganisationData = {

```

Comment:
There is already a `mock-organisation` object defined in the test folder, can we not use that?

Reply:
moved to ticket https://eaflood.atlassian.net/browse/PAY-2364